_literate/5_MCMC.jl

```diff
@@ -1239,8 +1239,8 @@ savefig(joinpath(@OUTPUT, "traceplot_bad_chain.svg")); # hide
 # If your Bayesian model has problems with convergence there are some steps that can be tried[^QR].
 # Listed here from the simplest to the most complex:
 
-# 1. **Increase the number of iterations and chains**: First option is to increase the number of MCMC iterations and it is also possible to increase the number of paralle chains to be sampled.
-# 2. **Model reparametrization**: the second option is to reparameterize the model. There are two ways to parameterize the model: the first with centered parameterization (CP) and the second with non-centered parameterization (NCP). NCP is most useful in Multilevel Models, therefore we will cover NCP in [10. **Multilevel Models**](/pages/10_multilevel_models/).
+# 1. **Increase the number of iterations and chains**: First option is to increase the number of MCMC iterations and it is also possible to increase the number of parallel chains to be sampled.
+# 2. **Model reparametrization**: the second option is to reparametrize the model. There are two ways to parameterize the model: the first with centered parametrization (CP) and the second with non-centered parameterization (NCP). NCP is most useful in Multilevel Models, therefore we will cover NCP in [10. **Multilevel Models**](/pages/10_multilevel_models/).
```

Comment:
```suggestion
# 2. **Model reparameterization**: the second option is to reparameterize the model. There are two ways to parameterize the model: the first with centered parameterization (CP) and the second with non-centered parameterization (NCP). NCP is most useful in Multilevel Models, therefore we will cover NCP in [10. **Multilevel Models**](/pages/10_multilevel_models/).
```

I think we should use reparameterize and reparameterization, because the original is parameterize and parameters.
I would like to keep the "e".